integration_tests/mockApis/mockResponses.js

```diff
@@ -1,4 +1,4 @@
-import searchResponse from '../../fake-manage-recalls-api/stubs/__files/search.json'
+import searchResponse from '../../fake-manage-recalls-api/stubs/__files/get-prisoner.json'
```

Comment:
You'll need to set up a new wiremock stub for the integration tests in https://github.com/ministryofjustice/manage-recalls-ui/blob/main/integration_tests/mockApis/manageRecallsApi.js, I think